src/js/actions/conversation-actions.js

```diff
@@ -54,3 +72,10 @@ export function resetUnreadCount() {
         type: RESET_UNREAD_COUNT
     };
 }
+
+export function setFetchingMoreMessagesFromServer(value) {
```

Comment:
Do we need this distinct event? Could we reuse the `setFetchingMoreMessages ` app state action?